packages/polymath-js/src/contracts/Contract.js

```diff
@@ -70,30 +111,21 @@ export default class Contract {
     ).eth.Contract(this._artifact.abi, this.address);
   }
 
-  /** @private */
-  _init(at?: Address) {
+  /**
+    Sets address and instantiates web3 contract
```

Comment:
Not too relevant since this whole library will be replaced, but just for the future. Usually comments like these are done in this format:

```suggestion
    /**
     * Sets address and instantiates web3 contract
     * ... stuff
     */
```

Reply:
My editor doesn't do it 😢 

I did it by hand though